_posts/2025-10-21-micrometer-prometheus-v1.adoc

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: 'Quarkus Micrometer using Prometheus client v1'
+date: 2025-06-12
+tags: micrometer prometheus
+synopsis: Micrometer has moved to use the Prometheus client v1. This post explains the migration path and required changes for applications.
+author: brunobat
+---
+
+== Quarkus Micrometer using Prometheus client v1
+
+Micrometer adopted Prometheus Client v1.x starting with version 1.13. However, Quarkus still uses Prometheus Client v0.x by default through the `quarkus-micrometer-registry-prometheus` extension which has been using the legacy `io.prometheus:simpleclient`.
+This is now considered *deprecated* and will not be part of a future Quarkus 4.x version (still no timeline).
+
+In parallel, we have introduced a new extension in Quarkiverse: https://docs.quarkiverse.io/quarkus-micrometer-registry/dev/micrometer-registry-prometheus-v1.html[`quarkus-micrometer-registry-prometheus-v1`], which integrates the new v1.x client.
+
+[INFO]
+====
+Users should move to the new `quarkus-micrometer-registry-prometheus-v1` extension.
```

Comment:
Hold on, does this change the metrics? If so, sure they should move but be aware that they dashboards and alerts and queries are now incorrect

Reply:
done